### PR TITLE
Cypress test to check if image was scaled to ROI

### PR DESCRIPTION
**Pull request by dvkruchinin**
Cypress test to check if image was scaled to ROI

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/case_7_image_scale_roi.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            cy.get('.cvat-resize-control')
+            .click()
+            cy.get('.cvat-canvas-container')
+            .trigger('mousedown', 309, 431, {which: 1})
+            .trigger('mousemove', 616, 671)
+            .trigger('mouseup', 616, 671)
+        })
+        it('Image scaled to ROI', () => {
+            cy.get('#cvat_canvas_background')
+            .should('have.attr', 'style').and('contain', 'scale(3.2817') //Chrome: scale(3.28173), Firefox: scale(3.28172)
```
Comment (bsekachev) [inline tests/cypress/integration/case_7_image_scale_roi.js:43]: I suppose this value depends on browser implementation and can be changed itself. Let's just check that scale is equal to 1 before and not equal 1 after.

**Review by bsekachev — CHANGES_REQUESTED** [review]

File: tests/cypress/integration/case_7_image_scale_roi.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/integration/case_7_image_scale_roi.js]: Done. Made a check of the original scale and check that after scaling the scale does not contain the original.

Comment (bsekachev) [reply tests/cypress/integration/case_7_image_scale_roi.js]: Hmm, again, hardcoding this value it is not the best way I believe.
Can we get the value 1.065 in runtime and than check if it was increased after zoom?

Comment (dvkruchinin) [reply tests/cypress/integration/case_7_image_scale_roi.js]: Changed it to get the initial value and compare it with the final one. Thanks.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**